tests/TestPNL.py

```diff
@@ -6,24 +6,26 @@ class TestPNL(unittest.TestCase):
 
     # # Data simulation
     # np.random.seed(0)
-    # x = np.random.randn(1000, 1)
+    # x = np.random.randn(200, 1)
     # x = x / np.std(x)
-    # e = np.random.randn(1000, 1)
+    # e = np.random.randn(200, 1)
     # e = e / np.std(e)
     # y_1 = (x + x**3 + e)**2
     # y_2 = np.exp(x**2 + e)
+    # y_3 = (np.exp(x) + e) ** 3
     # np.savetxt(r"TestData/pnl_simulation_1.txt", np.hstack([x, y_1]), delimiter=',')
     # np.savetxt(r"TestData/pnl_simulation_2.txt", np.hstack([x, y_2]), delimiter=',')
+    # np.savetxt(r"TestData/pnl_simulation_3.txt", np.hstack([x, y_3]), delimiter=',')
 
     # Set the threshold for independence test
-    p_value_threshold = 0.1 # useless now but left
+    p_value_threshold = 0.5 # useless now but left
```

Comment:
this is the only concern I have.

Do we have enough evidence to claim something if we compare with threshold 0.5? This basically means you can have false positive error as large as 0.5?